app/services/create_github_repo_service/stats_sender.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+# Wrapper class over GitHubClassroom.statsd for creating repositories
+# This enables us to add extra stats based on the progress of the repository being created
+#
+class CreateGitHubRepoService
+  class StatsSender
+    class InvalidStatError < StandardError; end
+    attr_reader :exercise
+    def initialize(exercise)
+      @exercise = exercise
+    end
+
+    # Public: Method for reporting a stat to GitHubClassroom.statsd,
+    #         raises an InvalidStatError if appropriate method not found.
+    #
+    # stat - symbol for calling a stat.
+    #
+    # Returns the current value of increment from statsd.
+    def report(stat)
+      send(stat)
+    rescue NoMethodError
```

Comment:
Can this be designed in a way that doesn't use `send` and `rescue NoMethodError`? If we do development on any methods called by `send` here and we introduce a NoMethodError during development, it will get swallowed here and make debugging a pain.

It looks like we have a mapping from symbols to error messages. Can this be a case statement?